src/main/resources/avro/references.avdl

```diff
@@ -31,65 +31,43 @@ field. New client and server implementations should prefer the "graph" mode.
 record Reference {
 
   /**
-  The `Segment` describing this `Reference`'s sequence in the `ReferenceSet`'s
+  The `Segment` describing this `Reference`'s `Sequence` in the `ReferenceSet`'s
   sequence graph.
 
-  The `Segment` will generally cover the entirety of the sequence, but may be
-  shorter if only a subrange of a sequence is to be included in the
+  The `Segment` will generally cover the entirety of the `Sequence`, but may be
```

Comment:
Amended after discussing with Adam: I'm suggesting removing `Segment` from `Reference` altogether, replacing it with an obligatory non-null `Sequence` ID combined with position and length information.